assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/mdsclient/DataCiteConnectionFactory.java

```diff
@@ -20,46 +22,49 @@
  * <p>Our {@link PasswordAuthentication} will only provide credentials for valid endpoints for the {@link HttpClient}
  * during server challenge (pre-emptive authentication).
  *
- * <p>Use {@link #getAuthenticatedConnection(URI)} to construct a new authenticated API connection.
+ * <p>Use {@link #getAuthenticatedMdsConnection(URI)}}
+ * or {@link #getAuthenticatedRestConnection(URI, DataCiteMdsClientSecretConfig)}
+ * to construct a new authenticated API connection.
  *
  * @see #createNvaCustomerAuthenticator(URI)
+
  */
-public class DataCiteMdsConnectionFactory {
+public class DataCiteConnectionFactory {
```

Comment:
Package should be `no.unit.nva.doi.datacite` as it no longer only a `mdsclient` factory.